crates/orderbook/src/api/get_order_status.rs

```diff
@@ -0,0 +1,31 @@
+use {
+    crate::orderbook::Orderbook,
+    anyhow::Result,
+    model::order::OrderUid,
+    shared::api::ApiReply,
+    std::{convert::Infallible, sync::Arc},
+    warp::{hyper::StatusCode, Filter, Rejection},
+};
+
+fn get_status_request() -> impl Filter<Extract = (OrderUid,), Error = Rejection> + Clone {
+    warp::path!("v1" / "status" / OrderUid).and(warp::get())
+}
+
+pub fn get_status(
+    orderbook: Arc<Orderbook>,
+) -> impl Filter<Extract = (ApiReply,), Error = Rejection> + Clone {
+    get_status_request().and_then(move |order_uid| {
```

Comment:
Maybe document in the openapi spec. But if we do, emphasis that the format might change at any time since we don't want to worry about stability on this endpoint.

Reply:
That would be great, specially since we use the openapi spec to generate the TS types in the SDK.

Reply:
Added OpenAPI doc.